addon/components/bs-form/element.js

```diff
@@ -25,6 +25,6 @@ export default class ValidatedBsFormElement extends BsFormElement {
   warnings;
 
   setupValidations() {
-    defineProperty(this, '_attrValidations', readOnly(`model.validations.attrs.${this.property}`));
+    defineProperty(this, '_attrValidations', readOnly(`args.model.validations.attrs.${this.args.property}`));
   }
```

Comment:
I would recommend to use a native getter instead. But didn't wanted to change to much in this hotfix.

```js
get _attrValidations() {
  const { model, property } = this.args;
  return get(model, `validations.attrs.${property}`);
}
```

Need to use `get` because `property` may be a path. We didn't decided yet if we treat as officially supported (https://github.com/kaliber5/ember-bootstrap/issues/1173) but changing it caused immediate bug reports in `ember-bootstrap-changeset-validations` (https://github.com/kaliber5/ember-bootstrap-changeset-validations/pull/28).

Reply:
When using the getter approach, wouldn't that allow us to easily support both ember-bootstrap <4.5 and >=4.5, by something like this:

```
get _attrValidations() {
  const model = this.args.model ? this.args.model : this.model;
  const property ? this.args.property ? this.args.property : this.property;
  return get(model, `validations.attrs.${property}`);
}
```

Or would the getter not interoperate well with the CP-world used in e-b <4.5? 🤔

Reply:
Yes. Such a getter should support both versions.

But that could be also done even without refactoring to the getter:

```js
    if (typeof this.args === 'object') {
      defineProperty(this, '_attrValidations', readOnly(`args.model.validations.attrs.${this.args.property}`));
    } else {
      defineProperty(this, '_attrValidations', readOnly(`model.validations.attrs.${this.property}`));
    }
```

Reply:
Ok, so...
* if supporting all ember-bootstrap versions is as easy as it *seems*, I think we should do that
* when we have to touch the code anyway, I would rather do the refactoring towards a getter

Reply:
I tried to refactor to this getter but reactivity was not working as expected. I guess you were right that the getter is not working well with the computed properties used in Ember Bootstrap < 4.4.

Reply:
Added support for Ember Bootstrap v4.4 back in 379f60f3d69400746aaa6078e503cc627119950c. Still using the `defineProperty` approach. Decided to not introduce `@embroider/macros` but use a simple if guard as the run-time overhead should not be relevant.